tox.ini

```diff
@@ -29,5 +29,12 @@ commands = pytest tests/ui/test_discopane.py {posargs}
 deps = flake8==3.3.0
 commands = flake8 {posargs:.}
 
+[testenv:dennis-lint]
+deps = dennis==0.9
+# Run via `bash -c` to make sure the file globbing are properly resolved on the bash
+commands = bash -c 'dennis-cmd lint --errorsonly locale/*/LC_MESSAGES/*.po'
```

Comment:
This won't check the pot files? Should they be validated too? E.g. https://github.com/mozilla/addons-frontend/tree/master/locale/templates/LC_MESSAGES

Reply:
Fair point, we never validated the .pot files in olympia but might be a good idea. I'll add it here, won't hurt I guess.

Reply:
Yeah the .pot files should be good, but this should hopefully help us in a case where the extraction process does something odd.